test/python/utils.py

```diff
@@ -20,11 +28,271 @@ def tarantool_version():
+        'fileusername', 'filegroupname'
+    ]
+
+    rpm = rpmfile.open(filename)
+    for key in expected_tags:
+        assert key in rpm.headers
+
+    for i, basename in enumerate(rpm.headers['basenames']):
+        # get filepath
+        basename = basename.decode("utf-8")
+        dirindex = rpm.headers[DIRINDEXES_TAG][i]
+        dirname = rpm.headers[DIRNAMES_TAG][dirindex].decode("utf-8")
+
+        filepath = os.path.join(dirname, basename)
+
+        # check fileowner
+        assert rpm.headers['fileusername'][i].decode("utf-8") == 'root'
+        assert rpm.headers['filegroupname'][i].decode("utf-8") == 'root'
+
+        # check filemodes
+        if filepath.startswith(os.path.join('/usr/share/tarantool/', project_name, '.rocks')):
+            continue
+
+        filemode = rpm.headers['filemodes'][i]
+        assert_filemode(filepath, filemode)
+
+
+def assert_tarantool_dependency_deb(filename):
+    with open(filename) as control:
+        control_info = control.read()
+
+        depends_str = re.search('Depends: (.*)', control_info)
+        assert depends_str is not None
+
+        min_version = re.findall(r'\d+\.\d+\.\d+', tarantool_version())[0]
+        max_version = str(int(re.findall(r'\d+', tarantool_version())[0]) + 1)
+
+        deps = depends_str.group(1)
+        assert 'tarantool (>= {})'.format(min_version) in deps
+        assert 'tarantool (<< {})'.format(max_version) in deps
+
+
+def assert_tarantool_dependency_rpm(filename):
+    rpm = rpmfile.open(filename)
+    dependency_keys = ['requirename', 'requireversion', 'requireflags']
+    for key in dependency_keys:
+        assert key in rpm.headers
+
+    assert len(rpm.headers['requirename']) == 2
+    assert len(rpm.headers['requireversion']) == 2
+    assert len(rpm.headers['requireversion']) == 2
+
+    min_version = re.findall(r'\d+\.\d+\.\d+', tarantool_version())[0]
+    max_version = str(int(re.findall(r'\d+', tarantool_version())[0]) + 1)
+
+    assert rpm.headers['requirename'][0].decode('ascii') == 'tarantool'
+    assert rpm.headers['requireversion'][0].decode('ascii') == min_version
+    assert rpm.headers['requireflags'][0] == 0x08 | 0x04  # >=
+
+    assert rpm.headers['requirename'][1].decode('ascii') == 'tarantool'
+    assert rpm.headers['requireversion'][1].decode('ascii') == max_version
+    assert rpm.headers['requireflags'][1] == 0x02  # <
+
+
+def check_systemd_dir(basedir):
+    systemd_dir = (os.path.join(basedir, 'etc/systemd/system'))
+    assert os.path.exists(systemd_dir)
+
+    systemd_files = recursive_listdir(systemd_dir)
+
+    assert len(systemd_files) == 2
+    assert '{}.service'.format(project_name) in systemd_files
+    assert '{}@.service'.format(project_name) in systemd_files
+
+
+def check_package_files(basedir, project_path):
+    # check if only theese files are delivered
+    for filename in recursive_listdir(basedir):
+        assert any([
+            filename.startswith(prefix) or prefix.startswith(filename)
```

Comment:
why?

Reply:
Because `recursive_listdir` result contains smth like `[..., /usr, /usr/share, /usr/share/tarantool/, /usr/share/tarantool/myapp ...]`.

Reply:
You can use "substring in string"

Reply:
But I want to check absolute filepaths.
What about smth like `filename = '/tmp/whoops/usr/share/tarantool/myapp'`? 
`'/usr/share/tarantool/myapp' in filename` will return `true`.

Reply:
Why do you use relpath in recursive listdir?

Reply:
Because files are unpacked to some temporarily directory in tests.

After package installation they will be unpacked to the `/` dir, and absolute path will be `/<relpath>`.